tests/core/test_cache.py

```diff
@@ -2,48 +2,37 @@
 
 import pytest
 
-from lnbits.cache import Cache
-from tests.conftest import pytest_asyncio
-
-
-@pytest_asyncio.fixture(scope="session")
-async def cache():
-    cache = Cache()
-
-    task = asyncio.create_task(cache.invalidate_forever(interval=0.1))
-    yield cache
-    task.cancel()
```

Comment:
I wouldn't remove this fixture, the invalidation needs go faster so that it actually runs in the expiration test - this doesn't show because the expiration is just done by the `get` call now, but the test can be adjusted to show that

Reply:
thats the only way how i could get rid of the task canceled error.
can you do that?

Reply:
weirdly the tests passed locally for me. maybe we a function with `cache.interval(0.1)` or something like that to dynamically set the cache interval?